### PR TITLE
Feature/game class

### PR DESCRIPTION
**Pull request by danalvrz**
- Created a Game class in a separate.rb file.
- Created an Author class with an association to the Item class (in a separate.rb file).
- All Game class properties visible in the diagram are defined and set up in the constructor method.
- All Author class properties visible in the diagram are defined and set up in the constructor method.
- Implement methods: add_item method in the Author class, and can_be_archived?() in the Game class
- Added unit tests for all implemented methods.

### CODE REVIEW

File: Classes/author.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    @items << item
+    item.author = self
+  end
+end
+
+# TEST **** #
+# game1 = Game.new('2000-01-01', true, '2010-01-01')
+# author1 = Author.new('richard', 'tonata')
+# author1.add_item(game1)
+# p game1.author.items
```
Comment (rtonata88) [inline Classes/author.rb:23]: To keep your code clean please make sure all to delete your test comments from the code.

File: Classes/game.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  def can_be_archived?
+    return true if super && (Date.today - @last_played_at > (365 * 2))
+
+    false
+  end
+end
+
+# **TEST**
+# game1 = Game.new('2021-01-01', true, '2020-01-01')
+# p game1.can_be_archived?
```
Comment (rtonata88) [inline Classes/game.rb:21]: To keep your code clean please make sure all to delete your test comments from the code.

**Review by rtonata88 — APPROVED** [review]
### STATUS: APPROVED 👍 
You have outdone yourself on this project @danalvrz. Keep up the good work. 

Just a few optional changes in the comments below and you are good to go.

Happy Coding

File: Classes/author.rb (hunk 1 above)
Comment (danalvrz) [reply Classes/author.rb]: done

File: Classes/game.rb (hunk 2 above)
Comment (danalvrz) [reply Classes/game.rb]: done

**Pull request merged**